proposals/stage-0/SECURITY_REPORTING/REQUIREMENTS.md

```diff
@@ -0,0 +1,18 @@
+# Minimum security reporting requirements for OpenJS Foundation projects
+
+## Security policy
+
+Each OpenJS Foundation project must publish a security policy in an easily accessible place. The recommended approach is to [publish the security policy in each GitHub repository](https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository).
+
+The [Node.js Ecosystem Security WG Template](https://github.com/nodejs/security-wg/blob/master/processes/responsible_disclosure_template.md) can be used by projects that do not have their own security reporting infrastructure.
+
+## Reporting
+
+Project security policy should explain how to confidentially report a security vulnerability.
+
+Each project should support at least one security reporting channel. Common ways of accepting vulnerability reports are:
+
+1. Designated email address, e.g. `security@example.com`.
```

Comment:
Is it worth mentioning that [/.well-known/security.txt](https://securitytxt.org/) can help make your security contact email discoverable.

Reply:
That would only be relevant for projects that have their own web sites?

Reply:
Fair point.  I suppose the way GH pages is laid out, *my-org-name.github.io/.well-known/security.txt* is not under organization control, so you'd have to pay for a CNAME.

Reply:
Yeah. I hope at some point we'll be able to provide a centralized `security@openjsf.org` email and then `https://openjsf.org/.well-known/security.txt` would be a good way to advertise it.